src/account/getAccountDetails.test.ts

```diff
@@ -0,0 +1,14 @@
+import { getAccountDetails } from './getAccountDetails';
+
+describe('Get account details', () => {
+
+    test('Verify that a account id is not null.', async () => {
+        const data = await getAccountDetails();
```

Comment:
Are you going to create separate functions for each action? For example, `getAccountDetails`, `getMovieList`, `getComedyMovies`, `getComedyCartoons`, etc? Isn't your code going to be repetitive while all those functions call `httpRequest` method you've created?